src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -278,11 +291,15 @@ protected Object decode(
                                 case 42:
                                 case 43:
                                     value = buf.readUnsignedShortLE();
-                                    position.set("fuel" + (j + 2 - 38), (value < 65500) ? value : null);
+                                    if (value < 65500) {
+                                        position.set("fuel" + (j + 2 - 38), value);
+                                    }
```

Comment:
Why are we changing all these?

Reply:
It seemed to me so clearer and removed null. You wrote earlier that you shouldn't set parameters to null if the check fails.

Reply:
When you have a lot of parameters, one liners are easier to read. My previous comment was about just assigning it to null. That just doesn't make any sense. What we had here made sense because it's a conditional check.

Reply:
Understood, return the previous version?